src/pages/rekey-reason/rekey-reason.reducer.ts

```diff
@@ -0,0 +1,119 @@
+import { RekeyReason } from '@dvsa/mes-test-schema/categories/B';
+import {
+  RekeyReasonActionTypes,
+  REKEY_REASON_IPAD_ISSUE_SELECTED,
+  REKEY_REASON_IPAD_ISSUE_TECH_FAULT,
+  REKEY_REASON_IPAD_ISSUE_LOST,
+  REKEY_REASON_IPAD_ISSUE_STOLEN,
+  REKEY_REASON_IPAD_ISSUE_BROKEN,
+  REKEY_REASON_TRANSFER_SELECTED,
+  REKEY_REASON_TRANSFER_STAFF_UPDATED,
+  REKEY_REASON_OTHER_SELECTED,
+  REKEY_REASON_OTHER_REASON_UPDATED,
+} from './rekey-reason.actions';
+
+export const initialState: RekeyReason = {
+  ipadIssue: {
+    selected: false,
+    broken: false,
+    lost: false,
+    technicalFault: false,
+    stolen: false,
+  },
+  other: {
+    selected: false,
+    reason: '',
+  },
+  transfer: {
+    selected: false,
+  },
+};
+
+export function officeReducer(state = initialState, action: RekeyReasonActionTypes): RekeyReason {
```

Comment:
This reducer has to be in the `tests` module because it's part of the test data that we collect throughout the journey. The page level reducer contains properties that are relevant only for the state of the page (i.e. `isLoading` state).

The reducer that you need to extend has already been created, it's called `rekeyReducer`. Currently it is a boolean variable only, but with your changes it will turn into an object encapsulating `isRekey` property and all the fields that you defined down below.